oclint-xcodebuild

```diff
@@ -185,6 +190,7 @@ def convert(input_file, output_file, is_excluded=None):
 def main():
     arg_parser = argparse.ArgumentParser(description='Converts xcodebuild logs or xctool-json-reporter logs to compile_commands.json')
     arg_parser.add_argument("-e", "-exclude", dest="exclusion", help="Directory exclusion pattern (regular expression)")
+    arg_parser.add_argument("-f", "--exclude-file", dest="file_exclusion", help="File exclusion pattern (regular expression)")
```

Comment:
Not sure if the long option for the exclude-folder is valid. I'll take a look on Monday.